mods/hv/rules/buildings.yaml

```
@@ -2036,7 +2036,6 @@ VGATE:
 
 WALLT:
 	Inherits: ^Wall
-	Inherits@Shape: ^1x1Shape
 	Building:
 		Dimensions: 1,1
 		Footprint: =
```

```suggestion
		Footprint: x
```
Just disallow units hiding there.

---

Okay!